cmd/cvp-trigger/main.go

```diff
@@ -80,20 +201,33 @@ func main() {
 		logrus.WithError(err).Fatal("failed to read Prow configuration")
 	}
 
-	var selected *prowconfig.Periodic
-	for _, job := range config.AllPeriodics() {
-		if job.Name == o.periodic {
-			selected = &job
-			break
-		}
+	prowjob, err := getProwJob(o.jobName, config)
+	if err != nil {
+		logrus.WithField("job-name", o.jobName).Fatal(err)
 	}
 
-	if selected == nil {
-		logrus.WithField("job-name", o.periodic).Fatal("failed to find the job")
+	// Add flag values to inject as ENV var entries in the prowjob configuration
+	envVars := map[string]string{
+		"OCP_VERSION":  o.ocpVersion,
+		"CLUSTER_TYPE": "aws",
+		"OO_INDEX":     o.indexImageRef,
```

Comment:
These have constants in the repo, let's import and use them

Reply:
I see the following:
``` go
const (
	ooIndex            = "OO_INDEX"
	ooPackage          = "OO_PACKAGE"
	ooChannel          = "OO_CHANNEL"
	ooInstallNamespace = "OO_INSTALL_NAMESPACE"
	ooTargetNamespaces = "OO_TARGET_NAMESPACES"
)
```
 in `pkg/steps/optional_operators.go` but they are un-exported. 
Should I make a change to export these?

Reply:
I wouldn't mind exporting these, but I'm wondering if we should centralize all these env names which can be set from outside (these, `RELEASE_*` ones, I don't know if there are more) to a single place.

Reply:
Let's just export them for now.

Reply:
I've updated these to be exported